Containerfile

```diff
@@ -0,0 +1,34 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
+
+# Maintainer details
+
+MAINTAINER <tzumainn@redhat.com>
+
+# The working directory for esi-leap inside the container
+
+WORKDIR /esi-leap
+
+# Copy the files of esi leap inside the container
+
+COPY . .
+
+# Run command to install dependencies for esi-leap
+
+RUN pip install esi-leap
+
+# Something for esi-leap-api and esi-leap-manager
+# Not sure how to link these two here / bring inside the container
+
+# Start processes for esi-leap-api and esi-leap-manager
+
+# What is the entry point / executable for esi-leap?
+
+CMD ["python", ./executable]
```

Comment:
Maybe use the commands here? https://github.com/CCI-MOC/esi-leap#run-the-services